<ui version="4.0" >
 <class>KPlato::PertEditor</class>
 <widget class="QWidget" name="KPlato::PertEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>200</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QTreeWidget" name="taskList" >
       <column>
        <property name="text" >
         <string>Tasks</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="available" >
       <column>
        <property name="text" >
         <string>Available tasks</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="addBtn" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip" >
          <string>Add required task</string>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeBtn" >
         <property name="maximumSize" >
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip" >
          <string>Remove required task</string>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="RelationTreeView" name="required" />
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RelationTreeView</class>
   <extends>QTreeView</extends>
   <header>kptrelationeditor.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>taskList</tabstop>
  <tabstop>available</tabstop>
  <tabstop>addBtn</tabstop>
  <tabstop>removeBtn</tabstop>
  <tabstop>required</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
